test/near_provider.test.js

```diff
@@ -662,3 +758,8 @@ async function getLatestBlockInfo () {
 async function waitForABlock () {
     return await new Promise((r) => setTimeout(r, 1000));
 }
+
+function isHex(h) {
```

Comment:
There is a `utils.isHex` method. The regex is slightly different. My regex is not the best...is one better than the other? If so, then can you update the method in `utils` and use that so we are using the same method everywhere?

`web3-utils` also has an `isHex` and `isHexStrict` method, but I didn't know that until after the fact. Perhaps at some point it would be best to switch over to using `isHexStrict`.